packages/paste-core/components/progress-bar/stories/customization.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {Label} from '@twilio-paste/label';
+import {HelpText} from '@twilio-paste/help-text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Form, FormControl} from '@twilio-paste/form';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {ProgressBar} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/ProgressBar/Customization',
+};
+
+const NumberFormatter = new Intl.NumberFormat('en-US');
+
+export const CustomizationWrapper: React.FC<React.PropsWithChildren<{isTestEnvironment: boolean}>> = ({
+  isTestEnvironment,
+}): React.ReactElement => {
+  const progressBarId = useUID();
+  const helpTextId = useUID();
+  const [value, setValue] = React.useState<number>(0);
+  const [rerun, setRerun] = React.useState<number>(1);
+
+  // Randomly updates the value of the progress bar to simulate a real progress bar
+  React.useEffect(() => {
+    let interval;
+    if (rerun) {
+      interval = setInterval(() => {
+        setValue((previousValue) => {
+          if (previousValue >= 100) {
+            setRerun(0);
+            return 100;
+          }
+          return previousValue + Math.random() * 12;
+        });
+      }, 600);
+    }
+    return () => clearInterval(interval);
+  }, [rerun]);
+
+  return (
+    <CustomizationProvider
+      disableAnimations={isTestEnvironment}
+      baseTheme="default"
+      elements={{
+        PROGRESS_BAR: {
+          backgroundColor: 'colorBackgroundPrimaryWeakest',
+        },
+        PROGRESS_BAR_FILL: {
+          borderRadius: 'borderRadius0',
+          backgroundColor: 'colorBackgroundBrand',
+        },
+
+        PROGGER: {
+          backgroundColor: 'colorBackgroundPrimaryWeakest',
+        },
+        PROGGER_FILL: {
+          borderRadius: 'borderRadius0',
+          backgroundColor: 'colorBackgroundBrand',
+        },
+      }}
+    >
+      <Box maxWidth="600px">
+        <Form>
+          <FormControl>
+            <Label htmlFor={progressBarId}>Downloading more ram</Label>
```

Comment:
issue: you're going to have to do what @nkrantz is doing in Meter, basically identically, because a Label for a `div` doesn't work.